ui_framework/src/global_styling/utilities/_typography.scss

```diff
@@ -0,0 +1,7 @@
+.uTextAlignRight {
```

Comment:
Dunno if I like the `u` prefix for utility classes. Can't think of anything I like better though.